engine/table/src/main/java/io/deephaven/engine/table/impl/by/PartitionByChunkedOperator.java

```diff
@@ -508,6 +586,7 @@ public void propagateInitialState(@NotNull final QueryTable resultTable) {
                 }
             }
         }
+        initialized = true;
```

Comment:
We should be throwing away `addedBuildersSequential` here if you don't accept my consolidation suggestion. We should not permanently waste that heap.